mayastor/src/rebuild/rebuild_impl.rs

```diff
@@ -42,9 +42,10 @@ struct TaskResult {
 }
 
 /// Number of concurrent copy tasks per rebuild job
-const SEGMENT_TASKS: usize = 4;
+const SEGMENT_TASKS: usize = 16;
 /// Size of each segment used by the copy task
-pub const SEGMENT_SIZE: u64 = 10 * 1024; // 10KiB
+/// Set to 64KiB to match SPDK's max buffer size (SPDK_BDEV_LARGE_BUF_MAX_SIZE)
+pub const SEGMENT_SIZE: u64 = 64 * 1024; // 64KiB
```

Comment:
Could we just set it to SPDK_BDEV_LARGE_BUF_MAX_SIZE then?

Reply:
Good point - done